xilem_web: Setting fill color on SVG stroke fails to compile

I have the following code snippet:
```rust
path.stroke(Color::BLACK, stroke_thin.clone()).fill(Color::TRANSPARENT)
```
Where `path` is a `kurbo::BezPath`. This fails to compile with the following message:

```
error[E0599]: the method `fill` exists for struct `Stroke<BezPath, _, _>`, but its trait bounds were not satisfied
   --> src/lib.rs:135:56
    |
135 |         path.stroke(Color::BLACK, stroke_thin.clone()).fill(Color::TRANSPARENT),
    |                                                        ^^^^ method cannot be called on `Stroke<BezPath, _, _>` due to unsatisfied trait bounds
    |
    = note: the following trait bounds were not satisfied:
            `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgGeometryElement<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgCircleElement<_, _>`
            `<&xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgCircleElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgCircleElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgGeometryElement<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgEllipseElement<_, _>`
            `<&xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgEllipseElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgEllipseElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: SvgGraphicsElement<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvggElement<_, _>`
            `<&xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvggElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvggElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: SvgElement<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgmPathElement<_, _>`
            `<&xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgmPathElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgmPathElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgGeometryElement<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgPolygonElement<_, _>`
            `<&xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgPolygonElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgPolygonElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgGeometryElement<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgPolylineElement<_, _>`
            `<&xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgPolylineElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgPolylineElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgGeometryElement<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgRectElement<_, _>`
            `<&xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgRectElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgRectElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: SvgGraphicsElement<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgTextContentElement<_, _>`
            `<&xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgTextContentElement<_, _>`
            `&xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgTextContentElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgGeometryElement<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgCircleElement<_, _>`
            `<&mut xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgCircleElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgCircleElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgGeometryElement<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgEllipseElement<_, _>`
            `<&mut xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgEllipseElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgEllipseElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: SvgGraphicsElement<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvggElement<_, _>`
            `<&mut xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvggElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvggElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: SvgElement<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgmPathElement<_, _>`
            `<&mut xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgmPathElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgmPathElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgGeometryElement<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgPolygonElement<_, _>`
            `<&mut xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgPolygonElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgPolygonElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgGeometryElement<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgPolylineElement<_, _>`
            `<&mut xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgPolylineElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgPolylineElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgGeometryElement<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgRectElement<_, _>`
            `<&mut xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgRectElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgRectElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: SvgGraphicsElement<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgTextContentElement<_, _>`
            `<&mut xilem_web::svg::Stroke<BezPath, _, _> as View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>>::Element = Pod<_, _>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgTextContentElement<_, _>`
            `&mut xilem_web::svg::Stroke<BezPath, _, _>: View<_, _, ViewCtx, Box<(dyn xilem_web::Message + 'static)>>`
            which is required by `&mut xilem_web::svg::Stroke<BezPath, _, _>: xilem_web::interfaces::SvgTextContentElement<_, _>`
```

The error message isn't very helpful. This code used to work on an earlier version of xilem_web but I'm not entirely sure when this broke. For more details on surrounding context, see https://github.com/linebender/gpu-stroke-expansion-paper/pull/46

--- a/xilem_web/src/interfaces.rs
+++ b/xilem_web/src/interfaces.rs
@@ -2004,9 +2004,6 @@ where
 pub trait SvgmPathElement<State, Action = ()>:
     SvgElement<State, Action, DomNode: AsRef<web_sys::SvgmPathElement>>
 {
-    fn fill(self, brush: impl Into<peniko::Brush>) -> crate::svg::Fill<Self, State, Action> {
-        crate::svg::fill(self, brush)
-    }
 }
 
 // #[cfg(feature = "SvgmPathElement")]
@@ -2021,6 +2018,9 @@ where
 pub trait SvgPathElement<State, Action = ()>:
     SvgGeometryElement<State, Action, DomNode: AsRef<web_sys::SvgPathElement>>
 {
+    fn fill(self, brush: impl Into<peniko::Brush>) -> crate::svg::Fill<Self, State, Action> {
+        crate::svg::fill(self, brush)
+    }
 }
 
 // #[cfg(feature = "SvgPathElement")]

